packages/util-rxjs/test/coldObservableProvider.test.ts

```diff
@@ -8,11 +9,24 @@ jest.mock('backoff-rxjs', () => ({
   retryBackoff: jest.fn().mockImplementation((...args) => jest.requireActual('backoff-rxjs').retryBackoff(...args))
 }));
 
+const createMockLogger = (): Logger => ({
+  debug: jest.fn(),
+  error: jest.fn(),
+  info: jest.fn(),
+  trace: jest.fn(),
+  warn: jest.fn()
+});
+
 describe('coldObservableProvider', () => {
   it('returns an observable that calls underlying provider on each subscription and uses retryBackoff', async () => {
     const underlyingProvider = jest.fn().mockResolvedValue(true);
     const backoffConfig: RetryBackoffConfig = { initialInterval: 1 };
-    const provider$ = coldObservableProvider({ provider: underlyingProvider, retryBackoffConfig: backoffConfig });
+    const logger = createMockLogger();
```

Comment:
logger object should be created in `beforeEach`

Reply:
done here: https://github.com/input-output-hk/cardano-js-sdk/commit/867945df0b58d37b9a1955b794e8950cfcf5b743